frontend/packages/dev-console/src/components/topology/D3ForceDirectedRenderer.tsx

```diff
@@ -425,22 +501,43 @@ export default class D3ForceDirectedRenderer extends React.Component<
       onSelect,
       topology,
     } = this.props;
-    const { nodes, edges, nodesById, edgesById, groups, groupsById, zoomTransform } = this.state;
+    const {
+      nodes,
+      edges,
+      nodesById,
+      edgesById,
+      groups,
+      groupsById,
+      zoomTransform,
+      dragNodeId,
+      sourceGroup,
+      targetGroup,
+    } = this.state;
     return (
-      <svg height={height} width={width} ref={this.refSvg} onClick={this.deselect}>
+      <svg
+        height={height}
+        width={width}
+        ref={this.refSvg}
+        onClick={this.deselect}
+        className={dragNodeId ? 'drag-active' : ''}
```

Comment:
We should qualify this class name:
```suggestion
        className={dragNodeId ? 'odc-m-drag-active' : ''}
```
Or `odc-m-dragging`